java/org/apache/tomcat/util/net/AbstractEndpoint.java

```diff
@@ -568,6 +569,14 @@ public final int getLocalPort() {
     protected abstract InetSocketAddress getLocalAddress() throws IOException;
 
 
+    /**
+     * Address for the unix domain socket.
+     */
+    private Path path;
```

Comment:
s/path/udsAddress/ ?!

Reply:
Since TCP sockets use `hostname`, path is fine. Already discussed, it should be `address` regardless of the family.

Reply:
The underlying API uses "path" as the reference to file that will become the socket:

https://linux.die.net/man/7/unix

```
#define UNIX_PATH_MAX    108

struct sockaddr_un {
    sa_family_t sun_family;               /* AF_UNIX */
    char        sun_path[UNIX_PATH_MAX];  /* pathname */
};
```

Naming it something else introduces new/inconsistent terminology.